modules/ROOT/pages/mule-app-dev-helloworld.adoc

```diff
@@ -0,0 +1,193 @@
+= Hello Mule Tutorial
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To demonstrate a simple HTTP request-response flow, this example responds
+with the message "Hello Mule" to requests submitted from a Web browser.
+The example demonstrates how a Mule application interacts with a user over an
+HTTP request.
+
+You set up and run this application as a project in Anypoint Studio. In a more
+complex application, it is possible to do much more than this simple example
+shows. For example, you can connect to external servers (such as SAP,
+Salesforce, a database, or an FTP server), import and implement API
+specifications directly within a Mule project, set up security, and process and
+transform the data in programmatic ways. It is beyond the scope of this
+example to explore such tasks.
+
+.hellomule in Studio
+image::hellomule.png[Hello Mule]
+
+== Prerequisites
+
+xref:7.3@studio::to-download-and-install-studio.adoc[Studio 7] is required.
```

Comment:
Remember not to specify a version unless it's really necessary (like if the process is not compatible with newer versions). 
This link will need to be updated when 7.4 is released.

Reply:
👍